Sources/StreamChat/Database/DTOs/CurrentUserDTO.swift

```diff
@@ -14,7 +14,8 @@ class CurrentUserDTO: NSManagedObject {
     /// Contains the timestamp when last sync process was finished.
     /// The date later serves as reference date for the last event synced using `/sync` endpoint
     @NSManaged var lastSynchedEventDate: DBDate?
-
+    
+    @NSManaged var blockedUserIds: [String]
```

Comment:
Should we do `Set<String>` here since the order is not important and Set avoids possible duplicates (although it should never happen)

Reply:
Updated 👍